src/api/types/TypeRegistrySpec.js

```diff
@@ -1,35 +1,50 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2017, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
 define(['./TypeRegistry', './Type'], function (TypeRegistry, Type) {
     describe('The Type API', function () {
-        var typeregistry;
+        var typeRegistryInstance;
 
         beforeEach(function () {
-            typeregistry = new TypeRegistry ();
-            typeregistry.addType('testtype', {
-                name: 'Test Type',
+            typeRegistryInstance = new TypeRegistry ();
+            typeRegistryInstance.addType('testType', {
+                label: 'Test Type',
                 description: 'This is a test type.',
                 creatable: true
             });
         });
-        it('new types are registered successfully', function () {
-            expect(typeregistry.types.testtype.definition.name).toBe('Test Type');
-        });
 
-        it('type registry contains new keys', function () {
-            expect(typeregistry.listKeys ()).toContain('testtype');
+        it('types can be standardized', function () {
+            typeRegistryInstance.standardizeType(typeRegistryInstance.types.testType);
+            expect(typeRegistryInstance.types.testType.definition.label).toBeUndefined();
+            expect(typeRegistryInstance.types.testType.definition.name).toBe('Test Type');
         });
 
-        it('types can be retrieved', function () {
-            expect(typeregistry.get('testtype').definition.description).toBe('This is a test type.');
+        it('new types are registered successfully and can be retrieved', function () {
+            expect(typeRegistryInstance.types.testType.definition.name).toBe('Test Type');
```

Comment:
get() is the API for retrieving a registered type which should be used instead of the`types` object. Something like this:

`
expect(typeRegistryInstance.get('testType').definition.description).toBe('This is a test type.');
`
or
`
expect(typeRegistryInstance.get('testType').definition.name).toBe('Test Type');
`